pynecone/utils.py

```diff
@@ -298,6 +299,44 @@ def get_config() -> Config:
         return Config(app_name="")
 
 
+def check_node_version(min_version):
+    """Check the version of Node.js.
+
+    Args:
+        min_version: The minimum version of Node.js required.
+
+    Returns:
+        Whether the version of Node.js is high enough.
+    """
+    try:
+        # Run the node -v command and capture the output
+        result = subprocess.run(
+            ["node", "-v"], stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
+        # The output will be in the form "vX.Y.Z", so we can split it on the "v" character and take the second part
+        version = result.stdout.decode().strip().split("v")[1]
+        # Split the version string on the "." character and convert each part to an integer
+        version_parts = [int(x) for x in version.split(".")]
+        # Split the minimum version string on the "." character and convert each part to an integer
+        min_version_parts = [int(x) for x in min_version.split(".")]
+        # Compare the version parts to the minimum version parts
+        if version_parts[0] < min_version_parts[0] or (
```

Comment:
I *think* you can simplify this to just
```
return version_parts > min_version_parts
```
Since Python should do an element-wise comparison on the lists.